modules/statics/src/coins/erc20Coins.ts

```diff
@@ -0,0 +1,44 @@
+import { UnderlyingAsset } from '../base';
```

Comment:
Is this meant for all erc20's or just eth erc20's?

Reply:
Intention-wise, we will be using this file for any eth erc20 tokens. Any already onboarded tokens can be migrated to this structure in future depending on resource availability.

Reply:
If we want to keep this file for only eth erc20 tokens, please rename it to something like `ethErc20Coins`

Reply:
Since ERC-20 tokens are built on the Ethereum blockchain, i don't think we need to explicitly mention eth. erc20 should be self-explanatory. 
Please update me if my understanding is wrong about all erc20 tokens being eth tokens.

Reply:
Once this PR is merged and the entire flow is tested on prod and we have taken care of concerns like how would the validation be done(easier rn to just go through one file) and do we need to update the onboarding docs etc, we can focus on migrating the existing ones.
I'll take the responsibility to create a tech-debt ticket in future once.